common/reflect_resource_test.go

```diff
@@ -609,15 +609,12 @@ func TestDataResource(t *testing.T) {
 				return fmt.Errorf("happens")
 			}
 			return nil
-		})
+		}).ToResource()
 	}()
 	d := r.TestResourceData()
 	d.Set("in", "test")
 
-	diags := r.ReadContext(context.Background(), d, nil)
-	assert.Len(t, diags, 1)
```

Comment:
This is an unnecessary test case, as the provider will ensure that the `meta` argument is always set to a non-nil `*DatabricksClient`.